node-playwright-chrome/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN \
     && apt-get install -y wget gnupg unzip ca-certificates xvfb --no-install-recommends \
     && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
     && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
-    && sh -c 'echo "deb http://ftp.us.debian.org/debian bullseye main non-free" >> /etc/apt/sources.list.d/fonts.list' \
+    && sh -c 'echo "deb http://ftp.us.debian.org/debian bookworm main non-free" >> /etc/apt/sources.list.d/fonts.list' \
```

Comment:
lol so this was the missing piece when i tried to do this myself yesterday :D

Reply:
One of them, the other one was forcing `--platform=linux/amd64` as otherwise the images would not build on M series SoCs